ECMA2Yaml/IntellisenseFileGen/Constants.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IntellisenseFileGen
+{
+    public class Constants
+    {
+        // [!INCLUDE[vstecmsbuild] (~/includes/vstecmsbuild-md.md)]
+        public static readonly string Include_Pattern1 = "(\\[!INCLUDE.*?\\((.*?)\\)\\])";
```

Comment:
for better performance, we should use something like `static Regex Include_Pattern1 = new Regex("(\\[!INCLUDE.*?\\((.*?)\\)\\])", RegexOptions.Compiled);`